product_docs/docs/eprs/6.2/eprs_rel_notes/index.mdx

```diff
@@ -2,12 +2,15 @@
 title: "Release Notes"
 redirects:
   - ../01_whats_new/
+navigation:
+  - eprs_rel_notes_6.2.19
 ---
 
 The EDB Postgres Replication Server documentation describes the latest version of EDB Postgres Replication Server 6.2 including minor releases and patches. The release notes in this section provide information on what was new in each release. For new functionality introduced in a minor or patch release, there are also indicators within the content about what release introduced the feature.
 
 | Version | Release Date | 
 | ------- | ------------ | 
+| [6.2.19](eprs_rel_notes_6.2.19)    | 2022 Dec 09 |
```

Comment:
The release date will need to change to correspond to the actual release date, whenever that ends up being.